src/containers/TransferForm/Form/Info.js

```diff
@@ -40,10 +47,12 @@ function Info({ onSubmit }) {
       </TransferFormLabel>
       <TransferFormInput
         id="amount"
-        type="text"
+        type="number"
+        min="0"
```

Comment:
@jeftarmascarenhas did you try to use numbers with `.` ? cause I tried once and it was not working for input type number 🤔